.fixtures.yml

```diff
@@ -6,6 +6,6 @@ fixtures:
     concat:  "https://github.com/puppetlabs/puppetlabs-concat.git"
     mongodb: "https://github.com/voxpupuli/puppet-mongodb.git"
     qpid:    "https://github.com/theforeman/puppet-qpid.git"
-    squid3:  "https://github.com/thias/puppet-squid3.git"
+    squid:   "https://github.com/voxpupuli/puppet-squid.git"
   symlinks:
```

Comment:
Take the opportunity to remove the redundant `symlinks` section?

Reply:
We should probably do this on all our modules at the same time to keep them consistent.